### PR TITLE
Stands with Ukraine

### PR DESCRIPTION
**Pull request by Nek-**
This PR adds information about the current situation of Ukraine. The goals are:
- Explain to everybody what is happening
- Add a link to a source of possible actions to take
- Showing support against oppression

### CODE REVIEW

**Review by derrabus — CHANGES_REQUESTED** [review]
Note: I'm expressing my personal point of view here. I'm not answering on behalf of the Doctrine project.

I'm very much in favor of showing support for the people of Ukraine. However, I disagree with this particular initiative. I don't believe we help Ukranians by boycotting Russian software. And well, I'd feel like a hypocrite if I called for a boycott of JetBrains products while actively using a JetBrains product for most of my work.

**Comment by Nek-** [comment]
There is a part about boycotting Russian software, yes. But the main part is about information and financial support for humanitarians.

**Comment by beberlei** [comment]
We have a section and banner on doctrine-project.org, managing our campaign centrally makes more sense since we have so many repos. I would close this therefore

**Comment by derrabus** [comment]
I don't know how many people regulaly visit the website. We might increase visibility if we linked to the campaign page on our website from our main repos (ORM, DBAL, …).

**Comment by Nek-** [comment]
@derrabus I can change the link to the doctrine-project website. 👍

**Comment by derrabus** [comment]
Yes, please.

**Comment by Nek-** [comment]
Done!

File: README.md [hunk 1]
```diff
@@ -3,6 +3,9 @@
 | [![Build status][3.0 image]][3.0] | [![Build status][2.12 image]][2.12] | [![Build status][2.11 image]][2.11] |
 | [![Coverage Status][3.0 coverage image]][3.0 coverage]| [![Coverage Status][2.12 coverage image]][2.12 coverage] | [![Coverage Status][2.11 coverage image]][2.11 coverage]  |
 
+
+[![SWUbanner](https://raw.githubusercontent.com/vshymanskyy/StandWithUkraine/main/banner2-direct.svg)](https://www.doctrine-project.org/stop-war.html)
```
Comment (derrabus) [inline README.md]: We shouldn't user their banner if we link to our own page.

Comment (Nek-) [reply README.md]: I saw nothing wrong to that, but well fixed.

**Review by derrabus — APPROVED** [review]

**Review by SenseException — APPROVED** [review]

**Pull request merged**